extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/Credentials.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.streaming.kinesis
+
+import com.amazonaws.auth.AWSCredentials
+
+case class Credentials(accessKeyId: String, secretKey: String) extends AWSCredentials {
```

Comment:
Hm, I wonder if that actually solves the problem since it still extends the AWS class. If the point is to pass access / secret key, that can be done by properties already right? so the point of this must be to enable other credential types? in which case it may be unavoidable to use the AWS class, and maybe fine given the nature of this module.